relay-server/src/actors/project.rs

```diff
@@ -64,6 +64,9 @@ pub struct ProjectConfig {
     /// Configuration for sampling traces, if not present there will be no sampling.
     #[serde(skip_serializing_if = "Option::is_none")]
     pub dynamic_sampling: Option<SamplingConfig>,
+    /// Configuration for operation breakdown. Will be emitted only if present.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub operation_name_breakdown: Option<Vec<String>>,
```

Comment:
proposing `{"breakdowns": {"span.op": ["http"]}}`

Reply:
I think considering that ops might not necessarily directly relate to breakdowns it might be better to make this a dictionary and let more than one op feed into a breakdown:

```json
"breakdowns": {
  "http": ["span.op:http"],
  "resource": ["span.op:resource.img", "span.op:resource.link"]
}
```

Reply:
Following up on this. I missed @untitaker's concern of splitting the spec string, so we can also make this:

```json
"breakdowns": {
  "http": { field: "span.op", values: ["http"] }
  "resource": { field: "span.op", value: ["resource.*"] }
}
```

Reply:
@jan-auer @untitaker @mitsuhiko @k-fish  So after considering all of your feedback, the configuration schema I've implemented will be something like:

```
"breakdowns": {
    "span_ops": {
        "type": "spanOperations",
        "matches": ["http", "db", "browser", "resource"],
    },
    "span_ops_2": {
        "type": "spanOperations",
        "matches": ["http", "db", "browser"],
    },
}
```

Then the `breakdowns` field will be emitted onto the event like so:

```
{
        "span_ops": {
            "lcp": {"value": 202.1},
            "ops.time.http": {"value": 2000000.0},
            "ops.time.resource": {"value": 100001.003},
            "ops.total.time": {"value": 2200001.003},
        },
        "span_ops_2": {
            "ops.time.http": {"value": 2000000.0},
            "ops.total.time": {"value": 2000000.0},
        },
}
```

This shape should be easy to extend with new options/fields; as per @jan-auer 's ask. And at the same time, potentially support for multiple span ops breakdowns will be specialized for various product usecases (@k-fish 's ask).